comictaggerlib/seriesselectionwindow.py

```diff
@@ -534,18 +534,20 @@ def update_row(self, row: int, series: ComicSeries) -> None:
         item.setData(QtCore.Qt.ItemDataRole.UserRole, series.id)
         item.setFlags(QtCore.Qt.ItemFlag.ItemIsSelectable | QtCore.Qt.ItemFlag.ItemIsEnabled)
 
-        item_text = str(series.start_year)
-        item = self.twList.item(row, 1)
-        item.setText(item_text)
-        item.setData(QtCore.Qt.ItemDataRole.ToolTipRole, item_text)
-        item.setFlags(QtCore.Qt.ItemFlag.ItemIsSelectable | QtCore.Qt.ItemFlag.ItemIsEnabled)
+        if series.start_year is not None:
+            item_text = f"{series.start_year:04}"
+            item = self.twList.item(row, 1)
+            item.setData(QtCore.Qt.ItemDataRole.ToolTipRole, item_text)
+            item.setData(QtCore.Qt.ItemDataRole.DisplayRole, series.start_year)
```

Comment:
Why the change from setText to setData

Reply:
I imagine it was where I was trying to break apart the display text from the sorting. Reverted.